packages/lit-dev-content/site/blog/feed.xml.njk

```diff
@@ -0,0 +1,36 @@
+---
+eleventyExcludeFromCollections: true
+---
+<?xml version="1.0" encoding="utf-8"?>
+<feed xmlns="http://www.w3.org/2005/Atom">
```

Comment:
I really don't know RSS, so... I trust you? :)

Reply:
Atom seems like the better format. For example, it supports multiple languages of the same article. Got the sense from reading around that it's more popular now.

It's also what Eleventy documents as its template, which 
I mostly copied here: https://www.11ty.dev/docs/plugins/rss/

Checked our output with https://validator.w3.org/feed/ too